core/store/src/db.rs

```diff
@@ -305,27 +367,56 @@ impl Database for RocksDB {
         }
         Ok(self.db.write(batch)?)
     }
+
+    fn get_snapshot(self: Pin<Arc<Self>>) -> Box<dyn ReadSnapshot> {
```

Comment:
Where's get_snapshot called. didn't see it the change

Reply:
not called, just adding it. Will revert from this PR